bin/build.sh

```diff
@@ -362,6 +365,24 @@ then
   puts "Built Corral"
 fi
 
+if [ ${BUILD_SYMBOOGLIX} -eq 1 ]
+then
+  puts "Building Symbooglix"
+
+  cd ${ROOT}
+  git clone https://github.com/symbooglix/symbooglix.git ${SYMBOOGLIX_DIR} 
```

Comment:
I think that `git clone --recursive` will clone all the submodules right away, and so you can avoid the two additional git commands below. Please try this out.